resources/themes/Night Mapping/style.qss

```diff
@@ -36,13 +36,6 @@ QWebView, QTextBrowser
     background-color: #ffffff;
 }
 
-QToolTip
-{
-    border: 1px solid #222;
```

Comment:
Please keep the border style, otherwise we end up with a 1px white border which is definitively a visual regression. Moving the background and text color definition to the palette.txt was a good idea.

You might also want to tweak the blend of gray theme (I've quickly checked it, it 'accidently' looks OK but I'd suggest you to make sure the e.g. toolbar button tooltips match the map tips the same way you did for night mapping.

You're one commit away from this being merged, nice! :)

Reply:
The problem is that the map tip color for the default theme (light gray) is not the same as the tooltip color (yellowish), and QPalette has no role for the Tooltip border, so we cannot have a fully statisfying solution.